package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "foreman_discovery",
+  "version": "1.0.0",
+  "description": "DESCRIPTION",
+  "main": "index.js",
+  "scripts": {
+    "lint": "tfm-lint --plugin -d /webpack",
+    "test": "tfm-test --plugin",
+    "test:watch": "tfm-test --plugin --watchAll",
+    "test:current": "tfm-test --plugin --watch",
+    "publish-coverage": "tfm-publish-coverage",
+    "stories": "tfm-stories --plugin",
+    "stories:build": "tfm-build-stories --plugin",
+    "create-react-component": "yo react-domain"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/theforeman/foreman_discovery.git"
+  },
+  "bugs": {
+    "url": "http://projects.theforeman.org/projects/foreman_dsicovery/issues"
+  },
+  "peerDependencies": {
+    "@theforeman/vendor": ">= 4.0.2"
+  },
+  "dependencies": {
+    "react-intl": "^2.8.0"
```

Comment:
is this actually needed? it is in foreman core

Reply:
yeah I don't think we need it, and at least for now we don't use it.. :)